performant-software/DM
# Change wording 'annotations' -> 'links to'

### What does this PR do?
Changes the terminology from "annotations" to "links to" in the annotation list header and other instances within the rollover window.

### What issues does it address?
Closes #158 

### How to test
On [staging](http://dm-staging.middell.net/), open a resource that has a few links. Hover over each to bring up the rollover window. The loading text before the list is populated should say "finding links." The first header in the list of links and references should say "links to:", unless there are no links from the resource, in which case it should say "no links" or "resource has no links" if there are also no references to the resource
